sys/winnt/Makefile.msc

```diff
@@ -348,8 +347,8 @@ OBJS   = $(MDLIB) \
 	$(VOBJ11) $(VOBJ12) $(VOBJ13) $(VOBJ14) $(VOBJ15) \
 	$(VOBJ16) $(VOBJ17) $(VOBJ18) $(VOBJ19) $(VOBJ20) \
 	$(VOBJ21) $(VOBJ22) $(VOBJ23) $(VOBJ24) $(VOBJ25) \
-	$(VOBJ26) $(VOBJ27) $(VOBJ28) $(VOBJ29) $(VOBJ30) \
-	$(REGEX)  $(CURSESOBJ)
+	$(VOBJ26) $(VOBJ27) $(VOBJ29) $(VOBJ30) $(REGEX)  \
```

Comment:
What happened to VOBJ28?

Reply:
The only item in it got shifted to `VOBJ27` when I removed `extralev`

Reply:
Oh wait is that right? Now I'm confused lol

Reply:
It is unused but so are `VOBJ29` and `VOBJ30` -- I don't know whether `VOBJ28` should be put back in there or `VOBJ29` and `30` should be removed...

Reply:
I dunno why we're having this conversation in a pull request rather than irc :P